storage/register.go

```diff
@@ -0,0 +1,20 @@
+package storage
+
+import (
+	"github.com/onflow/flow-go/model/flow"
+	"io"
+)
+
+type Register interface {
+	RegisterReader
+	RegisterWriter
+	io.Closer
+}
+
+type RegisterReader interface {
+	Get(ID flow.RegisterID, height uint64) flow.RegisterEntry
+}
+
+type RegisterWriter interface {
+	Store(entry flow.RegisterEntry, height uint64) error
```

Comment:
```suggestion
	Store(registers []flow.RegisterEntry, height uint64) error
```

please add comments about indexing by height.

Reply:
you are right, sorry forgot to update in this isolated PR.